spec/abstract/Deterministic_A.thy

```diff
@@ -369,9 +242,9 @@ text \<open>The CDT in the implementation is stored in prefix traversal order.
   The following functions traverse its abstract representation here to
   yield corresponding information.
 \<close>
+(* FIXME: use Lib.hd_opt instead *)
```

Comment:
is this intended for this round of changes?

Reply:
Not really, this comment and the style changes to `next_child` were copied from rt.